cadet/cadet.py

```diff
@@ -95,8 +88,8 @@ def install_path_to_cadet_paths(install_path: Optional[os.PathLike]) -> (None, N
         dll_path = cadet_root / 'bin' / 'cadet.dll'
         dll_debug_path = cadet_root / 'bin' / 'cadet_d.dll'
     else:
-        dll_path = cadet_root / 'lib' / 'lib_cadet.so'
-        dll_debug_path = cadet_root / 'lib' / 'lib_cadet_d.so'
+        dll_path = cadet_root / 'lib' / 'libcadet.so'
+        dll_debug_path = cadet_root / 'lib' / 'libcadet_d.so'
```

Comment:
Apparently this was already faulty in the previous version in CADET-Python but should work now.